core/src/apps/cardano/address.py

```diff
@@ -134,6 +135,9 @@ def _path_to_staking_path(path: list) -> list:
 
 
 def _get_address_header(address_type: int, network_id: int) -> bytes:
+    if address_type == CardanoAddressType.BOOTSTRAP_ADDRESS:
+        raise ValueError("Bootstrap address does not contain an explicit header")
```

Comment:
I would add here a rather extensive comment on the matter (a full explanation of why bootstrap addresses always start with 0b1000 because of their cbor enconding). As it is now, this feature is not really documented anywhere and who knows who will read this and wonder in a couple of months.